packages/flutter_tools/templates/app_shared/windows.tmpl/runner/utils.cpp

```diff
@@ -45,17 +45,18 @@ std::string Utf8FromUtf16(const wchar_t* utf16_string) {
   if (utf16_string == nullptr) {
     return std::string();
   }
+  int source_length = static_cast<int>(wcslen(utf16_string));
```

Comment:
Instead of using `wcslen`, could we substract 1 from the `target_length`?

Reply:
Yes, this is the other approach but for some reason I thought getting source_length first looked more elegant.
But leaving -1 passed to WideCharToMultiByte() and subtracting 1 will avoid the extra counting. Let me change this.